store/tikv/coprocessor.go

```diff
@@ -235,6 +236,8 @@ func (r *copRanges) split(key []byte) (*copRanges, *copRanges) {
 	return r.slice(0, n), r.slice(n, r.len())
 }
 
+const rangesPerTask = 25000
```

Comment:
Better to add some comment about what this value is used to do, why need it, why set it to this value.